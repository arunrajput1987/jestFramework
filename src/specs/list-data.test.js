import formatList from '../app/list-data'

test('Verify list of input data', () => {
  const formattedList = formatList(
    'Star Wars Names',
    [
      {name: 'Arun'},
      {name: 'Seema'},
      {name: 'Anshu'},
      {name: 'Aryendra1'},
    ],
    'name'
  )
  expect(formattedList).toMatchSnapshot();
})
