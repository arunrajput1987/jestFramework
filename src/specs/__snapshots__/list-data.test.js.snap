// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Verify list of input data 1`] = `
"These are the items in the Star Wars Names:
  - Arun
  - Seema
  - Anshu
  - Aryendra1"
`;
